release/nightly_tests/dataset/data_ingest_benchmark.py

```diff
@@ -104,28 +141,48 @@ def make_ds(size_gb: int, parallelism: int = -1):
     return dataset
 
 
-def run_ingest_streaming(dataset_size_gb, num_workers):
+def run_ingest_streaming(dataset_size_gb, num_workers, use_gpu, early_stop):
     ds = make_ds(dataset_size_gb)
+    resources = {"num_cpus": 0.5}
+    if use_gpu:
+        resources["num_gpus"] = 0.5
```

Comment:
For my own understanding, why are we using 0.5 here?

Reply:
To be honest, I don't know either. I just followed the original resource "num_cpus=0.5". I think for the long term, we should probably use CPU/GPU memories here.